tasks/src/main/java/ru/sberbank/school/task02/util/Quote.java

```diff
@@ -1,14 +1,10 @@
 package ru.sberbank.school.task02.util;
 
+import lombok.*;
```

Comment:
Using the &#39;.*&#39; form of import should be avoided - lombok.*.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-79477160>*